src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -429,10 +438,19 @@ export class ExposureNotificationService {
       this.finalize();
     }
     const {keysFileUrls, lastCheckedPeriod} = await this.getKeysFileUrls();
+    captureMessage('keysFileUrls', keysFileUrls);
     try {
       captureMessage('lastCheckedPeriod', {lastCheckedPeriod});
-      await this.exposureNotification.provideDiagnosisKeys(keysFileUrls);
-      const exposureWindows = await this.exposureNotification.getExposureWindows();
+
+      let exposureWindows;
+      if (Platform.OS === 'android') {
+        exposureWindows = await this.exposureNotification.getExposureWindowsAndroid(keysFileUrls);
+      } else {
+        const summaries = await this.exposureNotification.detectExposure(exposureConfiguration, keysFileUrls);
+        exposureWindows = await this.exposureNotification.getExposureWindowsIos(summaries[0]);
```

Comment:
Is it possible that `summaries` is null, or doesn't have a length >=1?

Reply:
Yes it could have length zero. I'll add a check and will plan to revisit this in a future PR, since I think we should probably be feeding in all of our summaries to `getExposureWindowsIos`.